### ISSUE TITLE
No way to select post type or multiple taxonomies in the 'Posts' element

### ISSUE THREAD

**Issue by HQRaja** [ThemeFuse/Brizy#878]
While editing any page or template with Brizy, when I add a 'Posts' element, it doesn't let me specify the post type in its settings - I can only specify a taxonomy.

Interestingly, if I use a taxonomy assigned to any other CPT (rather than the default Category/Tag taxonomy that's for the default Posts type), I can select that taxonomy and it works as expected. But this limits me to retrieving CPTs only based on taxonomies, and doesn't work for any CPT that doesn't have any taxonomies, or for any CPTs that may have taxonomies but for which I just want to retrieve all posts regardless of taxonomies.

It doesn't even let me select the default 'Post' type...all I can select is a taxonomy.

Also, there are no options to include (or exclude) multiple taxonomies...we can simply select one taxonomy for each Posts module.

This severely limits our options of displaying posts and CPTs the way we need.

**Comment by HQRaja** [comment]
Any update on this?

**Comment by HQRaja** [comment]
@bagrinsergiu Any update on this?

**Comment by codeclinic** [comment]
Yes, we really need to be able to filter the Post block output by post type (including CPT's) not just by the post category. There is already category filtration so hopefully filtering by post type shouldn't be too big an issue to resolve.
